network-api/networkapi/wagtail-l10n-customization/static/css/language_toggles.css

```diff
@@ -0,0 +1,51 @@
+form .l10n-hidden {
```

Comment:
Is this a copy from another file? Using the element type for CSS targeting seems overly specific.

Reply:
we want to be overly specific. The most specific rule wins.